spec/spec_helper.rb

```diff
@@ -13,6 +13,13 @@
 # it.
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+
+require "simplecov"
+
+SimpleCov.start do
+  add_filter "/spec/"
+end
```

Comment:
I just added the SimpleCov settings to the master.
BTW, it looks that we don't need to add `add_filter`. Is it necessary?

Reply:
@necojackarc 
When we don't add the filter, the spec files are used to calculate the coverage as follows.

![screen shot 2018-02-12 at 13 45 04](https://user-images.githubusercontent.com/949545/36084275-1839dd6e-0fff-11e8-9291-2ae44c9301c3.png)

So I think we need to do that.

Reply:
Cool - let's add the filter!